libs/designer/src/lib/ui/panel/panelTabs/parametersTab/index.tsx

```diff
@@ -140,8 +140,8 @@ const ParameterSection = ({
       const { viewModel } = newState;
       const parameter = nodeInputs.parameterGroups[group.id].parameters.find((param: any) => param.id === id);
       if (
-        (parameter?.type === constants.SWAGGER.TYPE.BOOLEAN && value.length === 1 && value[0]?.value === 'True') ||
```

Comment:
I dont understand why are we patching the value here for boolean parameters in forst place, who added this code initially?

Reply:
I added this code originally to fix https://github.com/Azure/LogicAppsUX/issues/2012, but you're right that this should be done in the serialization